pkg/controller/controller_instance_test.go

```diff
@@ -1136,6 +1226,137 @@ func TestPollServiceInstanceSuccessDeprovisioningWithOperationWithFinalizer(t *t
 	assertNumEvents(t, events, 1)
 }
 
+func TestSetInstanceCondition(t *testing.T) {
+	instanceWithCondition := func(condition *v1alpha1.InstanceCondition) *v1alpha1.Instance {
+		instance := getTestInstance()
+		instance.Status = v1alpha1.InstanceStatus{
+			Conditions: []v1alpha1.InstanceCondition{*condition},
+		}
+
+		return instance
+	}
+
+	newTs := metav1.Now()
+
+	condition := func(cType v1alpha1.InstanceConditionType, status v1alpha1.ConditionStatus, s ...string) *v1alpha1.InstanceCondition {
+		c := &v1alpha1.InstanceCondition{
+			Type:   cType,
+			Status: status,
+		}
+
+		if len(s) > 0 {
+			c.Reason = s[0]
+		}
+
+		if len(s) > 1 {
+			c.Message = s[1]
+		}
+
```

Comment:
why set to a time in the past?
set to a time in the past to make sure that the code updates this timestamp when the condition changes.